### ISSUE TITLE
Torch-RNN poetry

### ISSUE THREAD

**Issue by jpcreamer** [jcjohnson/torch-rnn#106]
Here is a poem I wrote with the assistance of your code. Sorry if this is the wrong place to share -- thought people might find it interesting. I trained it with Walt Whitman's Leaves of Grass, too small of a training set no doubt. Still, it produced a lot of interesting phrases and images (as well as the title, verbatim, set off from other text as a title in the output.) I found the best results turning down the temperature to .6 or so. Don't get the wrong idea -- this was not straight output, but a molding of favorite bits from many many pages of output into a poem. Thanks for this, amazing stuff!

-- John

Song of the Light

I sail, burning the day 
to the drum sounds of work and the fire endless.
This soul, every year, I whittle
with my life and many agains.

But now the winds gather and stall
and I am stranded on the island of my day and life.
The ground is white death.
I see trees and the dark

and myself laid out,
the sleeves of me, the burial, the sound of the earth, 
and the soul sound, the mourning.
I am nothing, a father to a midge

like all of us,
the hands and masters and dead.
Heaven and the past,
out on a breath.

But let me take the dare of bards:
I am the grass here and the children.
I will seed the fields of young.
I am the city of men and the perfect soul in the earth.

This is how I save myself,
and now, by this silent poem, you.
